curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-declarative-functions-with-es6.english.md

```diff
@@ -23,9 +23,9 @@ Refactor the function <code>setGear</code> inside the object <code>bicycle</code
 ```yml
 tests:
   - text: Traditional function expression was not used.
-    testString: assert(!getUserInput('index').match(/function/),'Traditional <code>function</code> expression was not used.');
+    testString: getUserInput => assert(!getUserInput('index').match(/function/),'Traditional <code>function</code> expression was not used.');
```

Comment:
@ValeraS This is just a question, so I know for the future.  Why do asserts use this getUserInput structure on some challenges and not on others?  When is this type of assert statement necessary vs not necessary?

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/blob/0dc0edcded1dd6aa2c57cc408b57ea0064e607a6/client/src/templates/Challenges/utils/frame.js#L85-L89

@Bouncey can give more clarification on it.

Reply:
Yeah, I had already looked at that section of code, but I was still confused why many of the challenges do not need to use getUserInput.

Reply:
`getUserInput` is/was a precursor to multi-file editing. The idea being that if a challenge had a css file as well as an HTML file, `getUserInput` would be a way of selecting the correct file to test. 

For now it returns the same value as what you would find in `code`. This same test could be written like:
```js
assert(
  !code.match(/function/),
  'Traditional <code>function</code> expression was not used.'
)
```

This is an area we need to discuss the architecture for, but we also need some agreement on how to handle multiple files.

Reply:
@Bouncey That was what I suspected (regarding the use of `code`), but I wondered why the challenges were just not using it. I think it would be great if we could make it consistent across all challenges.  Thanks for clearing things up.  Are you going to create a GitHub issue, so the architecture can be discussed/agreed upon?